protocol/daemons/flags/flags.go

```diff
@@ -112,6 +121,16 @@ func AddDaemonFlagsToCmd(
 		"Socket address for the price daemon to send updates to, if not set "+
 			"will establish default location to ingest price updates from",
 	)
+	cmd.Flags().Bool(
```

Comment:
nit: the method comment says `required flags to instantiate a server and client for price update`. Can we update the comment to be less specific to price update since this is shared now?